src/utils/index.ts

```diff
@@ -3,12 +3,12 @@
 // eslint-disable-next-line no-nested-ternary
 export const ROOT_URL = process.env.TARGET_ENV === 'prod'
   ? 'https://betmate-backend-prod.herokuapp.com'
-  : process.env.TARGET_ENV === 'dev'
+  : process.env.TARGET_ENV === 'dev' || true
     ? 'https://betmate-backend-dev.herokuapp.com'
```

Comment:
The next few changes will just be UI related, so I will be keeping this through the upcoming PRs